FinnUI/Sources/Recycling/ProjectUnits/ProjectUnitCell.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+import FinniversKit
+
+protocol ProjectUnitCellDelegate: AnyObject {
+    func projectUnitCell(_ projectUnitCell: ProjectUnitCell, didTapFavoriteButton button: UIButton)
+}
+
+class ProjectUnitCell: UICollectionViewCell {
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(withAutoLayout: true)
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        return stackView
+    }()
+
+    private lazy var remoteImageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.contentMode = .scaleAspectFill
+        imageView.layer.cornerRadius = 8
```

Comment:
i guess is better to use a `.spacing` constant, then if we change those constants, our radius will change accordingly